deployment/helm/polder/values.yaml

```diff
@@ -0,0 +1,15 @@
+# Default values for polder.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This is to override the chart name.
+nameOverride: "polder"
+fullnameOverride: "test-polder"
+
+# polder custom configuration
+mapboxToken: ""
+stacApi: "https://rx2hna9pbg.execute-api.eu-central-1.amazonaws.com"
```

Comment:
Those are links for WHIS, I would replace them with the default URL generated by the default helm deploy on kubernetes. (e.g. polder-eopi-stac)

Reply:
Done.